CODE_OF_CONDUCT.md

```diff
@@ -1,93 +1,71 @@
-# Citizen Code of Conduct
+市民行動規範
+1。目的
```

Comment:
Let's be sure to use proper [Markdown syntax](https://www.markdownguide.org/) and headers with the `#` notation. Are you new to Markdown? Can we help?